docs/content/src/learn/tutorials/saas-integrations/gmail/working-with-googlespreadsheet-service/README.md

```diff
@@ -0,0 +1,196 @@
+# Google Spreadsheet Connector Service
+
+Ballerina is an open-source programming language that empowers developers to integrate their system easily with the 
+support of connectors. In this guide, we are mainly focusing on Google Spreadsheet Connector Service. 
+
+The `wso2/gsheets4` module allows you to perform the following operations.
+
+- Create a new spreadsheet
+- Create a new worksheet
+- View a spreadsheet
+- Add values into a worksheet
+- Get values from worksheet
+- Retrieving values of a column
+- Retrieving values of a row
+- Adding value into a cell
+- Retrieving value of a cell
+- Retrieving value of a cell
+
+This example explains how to use Google Spreadsheet Connector to perform the above operations.
+
+You can find other integrations modules from [wso2-ballerina](https://github.com/wso2-ballerina) GitHub organization.
+
+## What you'll build
+
+This example explains how to use the Google Spreadsheet client to connect with the Google Spreadsheet Connector instance and perform the 
+following operations:
```

Comment:
Can we list these here?

Reply:
Changed it to point to earlier mentioned operations.